### ISSUE TITLE
White page error

### ISSUE THREAD

**Issue by ovidenov** [openeuropa/oe_multilingual#155]
Dear support, 
I would like to report the following error:

TypeError: trim(): Argument #1 ($string) must be of type string, null given in trim() (line 53 of /var/www/html/web/modules/contrib/oe_multilingual/modules/oe_multilingual_url_suffix/src/UrlSuffixTrait.php).

To reproduce: 
1) Make sure oe_multilingual is enabled
2) Url is something like https://knowledge4policy.ec.europa.eu/news/tropical-moist-forest-degradation-impacts-are-much-worse-previously-thought%E2%80%AF_enhttps://

Basically, after the suffix "_en", there is "https://" which causes a white screen.
